docs/guide/caching.md

```diff
@@ -73,3 +73,77 @@ options: {
 ```
 
 Since custom piece types extend this module, configuring it at project level configures all of them with a new default. It is still possible to override the `cache` option in the individual modules.
+
+## Advanced Cache Invalidation
+
+The advanced cache invalidation system allows caching of pages and pieces using `ETag` header.
```

Comment:
using `ETag` headers.

Reply:
The purpose of this feature is to generate `ETag` headers that allow the browser to check whether a page or piece has changed, reducing the amount of work done on the server. The actual caching is not performed in ApostropheCMS itself. To gain the full benefit of this strategy, you must implement a cache at the CDN level via a service such as CloudFlare, or do so in your reverse proxy server, e.g. `nginx`. Otherwise caching will only take place in individual web browsers.